challenge_4/python/CindyB/src/Challenge4.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from Tree import Tree
+from TreeNode import TreeNode
+
+input = [4,2,7,1,3,6,9]
+myTree = Tree(TreeNode(input[0]))
```

Comment:
We encourage the use of Python's PEP8 for style and formatting. In this case myTree would instead be my_tree. (https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)  No need to change this if you don't want to, but going forward it would be nice.